api/src/cleanup/cleanup.module.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright © 2025 Hexastack. All rights reserved.
+ *
+ * Licensed under the GNU Affero General Public License v3.0 (AGPLv3) with the following additional terms:
+ * 1. The name "Hexabot" is a trademark of Hexastack. You may not use this name in derivative works without express written permission.
+ * 2. All derivative works must include clear attribution to the original creator and software, Hexastack and Hexabot, in a prominent location (e.g., in the software's "About" section, documentation, and README file).
+ */
+
+import { Global, Module } from '@nestjs/common';
+
+import { CleanupService } from './cleanup.service';
+
+@Global()
+@Module({
+  providers: [CleanupService],
+  exports: [CleanupService],
+})
+export class CleanupModule {}
```

Comment:
I would recommend naming the module : ExtensionModule . So that we can add other services such as the cleanup service.

Reply:
Yes 100% ExtensionModule name give a better meaning about the role of the module, happy to apply the update :technologist: